docs/readmes/howtos/troubleshooting/update_certificates.md

```diff
@@ -0,0 +1,49 @@
+---
+id: update_certificates
+title: Update SSL certificates on Orchestrator/Access Gateway
+hide_title: true
+---
+# Update SSL certificates on Orchestrator/Access Gateway
+
+**Description:** This document mentions the steps to be followed to update SSL certificates on Orchestrator/Access Gateway files. This changes the rootCA certificate for node.
+
+**Environment:** Orchestrator in Kubernetes/AWS
+
+**Affected components:** AGW, Orchestrator
+
+**Configuration steps:**
+
+1. Orchestrator changes:
+
+    a. Opt to upgrade modules and plugins as part of their respective installation steps.
+
+    `terraform init –upgrade`
+
+    b. Create a new `rootCA` cert and private key, then sign a new SSL cert against that `rootCA` and move into `~/secrets/certs`. Other certificates don't need to change and you can copy into the folder.
```

Comment:
Here, I'd prefer either a reference to a document on how to do this, or to list out the commands.

Reply:
I detailed more commands and added more info in this KB like
-  Specify this is to update only rootCA and controller certificates(not application or admin_operator certificates) 
- Specify this KB is to update the certificates due to expiration and not other change in certificate like CN.